Library/Formula/otto.rb

```diff
@@ -18,164 +18,61 @@ class Otto < Formula
+    github.com/hashicorp/atlas-go b66e377f52e316206efe71abba20e276d8790d86
+    github.com/hashicorp/errwrap 7554cd9344cec97297fa6649b055a8c98c2a1e55
+    github.com/hashicorp/go-checkpoint e4b2dc34c0f698ee04750bf2035d8b9384233e1b
+    github.com/hashicorp/go-cleanhttp ce617e79981a8fff618bb643d155133a8f38db96
+    github.com/hashicorp/go-getter c5e245982bdb4708f89578c8e0054d82b5197401
+    github.com/hashicorp/go-multierror d30f09973e19c1dfcd120b2d9c4f168e68d6b5d5
+    github.com/hashicorp/go-version 2b9865f60ce11e527bd1255ba82036d465570aa3
+    github.com/hashicorp/hcl 197e8d3cf42199cfd53cd775deb37f3637234635
+    github.com/hashicorp/terraform 6d1d46c47ced8880ff4af5d48e49572c62fe7df6
+    github.com/hashicorp/vault 4fa678131a73a77cbac7cb456fb69950d3146ca6
+    github.com/hashicorp/yamux df949784da9ed028ee76df44652e42d37a09d7e4
+    github.com/jmespath/go-jmespath c01cf91b011868172fdcd9f41838e80c9d716264
+    github.com/jteeuwen/go-bindata a0ff2567cfb70903282db057e799fd826784d41d
+    github.com/kardianos/osext 29ae4ffbc9a6fe9fb2bc5029050ce6996ea1d3bc
+    github.com/mattn/go-isatty 56b76bdf51f7708750eac80fa38b952bb9f32639
+    github.com/mitchellh/cli cb6853d606ea4a12a15ac83cc43503df99fd28fb
+    github.com/mitchellh/iochan 87b45ffd0e9581375c491fef3d32130bb15c5bd7
+    github.com/mitchellh/colorstring 8631ce90f28644f54aeedcb3e389a85174e067d1
+    github.com/mitchellh/copystructure 6fc66267e9da7d155a9d3bd489e00dad02666dc6
+    github.com/mitchellh/go-homedir d682a8f0cf139663a984ff12528da460ca963de9
+    github.com/mitchellh/gox 39862d88e853ecc97f45e91c1cdcb1b312c51eaa
+    github.com/mitchellh/ioprogress 8163955264568045f462ae7e2d6d07b2001fc997
+    github.com/mitchellh/mapstructure 281073eb9eb092240d33ef253c404f1cca550309
+    github.com/mitchellh/panicwrap a1e50bc201f387747a45ffff020f1af2d8759e88
+    github.com/mitchellh/prefixedio 6e6954073784f7ee67b28f2d22749d6479151ed7
+    github.com/mitchellh/reflectwalk eecf4c70c626c7cfbb95c90195bc34d386c74ac6
+  ]
+  otto_deps.each_slice(2) do |x, y|
+    go_resource x do
+      url "https://#{x}.git", :revision => y
+    end
   end
 
-  go_resource "github.com/vaughan0/go-ini" do
-    url "https://github.com/vaughan0/go-ini.git",
-      :revision => "a98ad7ee00ec53921f08832bc06ecf7fd600e6a1"
+  go_resource "gopkg.in/flosch/pongo2.v3" do
+    url "https://gopkg.in/flosch/pongo2.v3.git",
+      :revision => "5e81b817a0c48c1c57cdf1a9056cf76bdee02ca9"
   end
 
   go_resource "golang.org/x/crypto" do
     url "https://go.googlesource.com/crypto.git",
-      :revision => "c8b9e6388ef638d5a8a9d865c634befdc46a6784"
+      :revision => "803f01ea27e23d998825ec085f0d153cac01c828"
   end
 
-  go_resource "gopkg.in/flosch/pongo2.v3" do
-    url "https://gopkg.in/flosch/pongo2.v3.git",
-      :revision => "5e81b817a0c48c1c57cdf1a9056cf76bdee02ca9"
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools.git",
+      :revision => "4ad533583d0194672e7d3bc6fb8b67c8e905d853"
   end
```

Comment:
however, the `gopkg.in/flosch/pongo2.v3` resource can be included in `otto_deps`